config/settings.yml

```diff
@@ -34,6 +34,10 @@ mobile_application_platform:
   client_key_path: spec/fixtures/mobile_application_platform/oauth.key
   sign_up_service_url: https://cerner.apps-staging.va.gov
   sign_up_service_provisioning_api_key: abcd1234
+  sts:
```

Comment:
You'll need to rebase to pull in my commit and then drop your change (`sts: mock: true`) when working through the merge conflict.